CHANGELOG.md

```diff
@@ -73,13 +81,6 @@
 
 - Update CHANGELOG.md for release 1.0.0-beta2. [\#81](https://github.com/openeuropa/code-review/pull/81) ([ademarco](https://github.com/ademarco))
 - OPENEUROPA-1722: Lock php\_codesniffer version on 1.x until our patches are updated. [\#80](https://github.com/openeuropa/code-review/pull/80) ([ademarco](https://github.com/ademarco))
-- OPENEUROPA-1722: Lock php\_codesniffer version until our patches are updated. [\#79](https://github.com/openeuropa/code-review/pull/79) ([brummbar](https://github.com/brummbar))
-- OPENEUROPA-1607: Use config classes for PHP-CS-Fixer configuration. [\#72](https://github.com/openeuropa/code-review/pull/72) ([drupol](https://github.com/drupol))
-- OPENEUROPA-1566: Core Review: Add DrupalPractice standard from coder. [\#71](https://github.com/openeuropa/code-review/pull/71) ([sergepavle](https://github.com/sergepavle))
-- Issue \#94: Update README. [\#70](https://github.com/openeuropa/code-review/pull/70) ([drupol](https://github.com/drupol))
-- OPENEUROPA-1575: Add the PHP Linter and re-organize the tasks in proper order. [\#67](https://github.com/openeuropa/code-review/pull/67) ([drupol](https://github.com/drupol))
-- OPENEUROPA-1573: Add JSONLint GrumPHP task. [\#59](https://github.com/openeuropa/code-review/pull/59) ([drupol](https://github.com/drupol))
-- OPENEUROPA-1521: Use PHP CS Fixer. [\#50](https://github.com/openeuropa/code-review/pull/50) ([drupol](https://github.com/drupol))
```

Comment:
Why all these deletions?

Reply:
Yeah...the changelog is not generated properly.